frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -998,9 +1010,9 @@ export function HandMarkedPaperBallot({
               <Contest
                 key={contest.id}
                 density={layoutDensity}
-                section={dualPhraseWithSlash(
-                  contest.section,
-                  localeContestsById?.[contest.id]?.section,
+                districtName={dualPhraseWithSlash(
+                  getContestDistrictName(election, contest),
+                  localeDistrictsById?.[contest.districtId]?.name,
```

Comment:
I maintained this dual phrase stuff but not sure if we want to keep it in future PRs @jonahkagan.